source/manual/alerts/RouterErrorRatioTooHigh.html.md

```diff
@@ -0,0 +1,35 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Router error ratio too high
+parent: "/manual.html"
+layout: manual_layout
+section: Pagerduty alerts
+---
+
+You can find the router request error rates on this dashboard:
+
+- [5xx Router Requests][router-5x https://grafana.eks.production.govuk.digital/d/uBEWkijnz/router-request-rates-errors-durations?orgId=1 x-request-rates-grafana]
+
+You can also view the 500+503 error rates across all applications on this dashboard:
+
+- [5xx app Requests][app-5xx-error-rates-grafana]
+
+## description
+
+This alert will fire when the ratio of requests in an error state are above the threshold configured in
+this config [file][https://github.com/alphagov/govuk-helm-charts/blob/main/charts/monitoring-config/rules/router.yaml]
```

Comment:
Describe the file for accessibility, instead of "this config file". (Helps if you're a screen reader user as it saves you having to follow the link to gain context)

Reply:
Have described the metric as it's currently written and linked to the config file in case it changes.